.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
+- repo: https://github.com/lorenzwalthert/pre-commit-hooks
+  rev: v0.0.0.9015
```

Comment:
Do we need to be strictly use only that revision? Would `master` be a problem? We could omit the mention of `autoupdate` then.

Reply:
I think pre-commit configuration works this way by design - see https://pre-commit.com/#using-the-latest-version-for-a-repository

Maybe @lorenzwalthert can confirm / clarify this though?